<launch>
	<arg name="veh"/>
	<arg name="src"/>
	<remap from="repeater_node/topic_in" to="/$(arg src)/publisher_node/topic"/>
	<remap from="repeater_node/topic_out" to="reapeater_node/topic"/>
	<node ns="$(arg veh)" pkg="lab03" type="repeater_node.py" name="repeater_node" output="screen">
	</node>
</launch>
